tests/test_api_versions.py

```
@@ -0,0 +1,104 @@
+            for p in included
+        ), "api.routers not included at root ''"
+
+
+@pytest.mark.django_db
+class TestApiVersionPaths:
+    @pytest.mark.skipif(not HAS_DRF, reason="DRF not installed in test environment")
+    @pytest.mark.parametrize("path", [
+        "/api/v1/",
+        "/api/v2/",
+    ])
+    def test_version_index_paths_exist_or_404_clean(self, path: str, db: Any) -> None:
```

Ruff (0.12.2)

Dynamically typed expressions (typing.Any) are disallowed in `db`

(ANN401)